account_invoice_report_service/static/src/scss/report_invoice_service_styles.scss

```
@@ -47,9 +47,13 @@
 
 .ris-table2 {
     @include ris-basetable-mixin;
-    border-bottom: 1px solid rgb(221, 221, 221);
 }
 
 .ris-table3 {
     @include ris-basetable-mixin;
 }
+
+.border-bottom > td {
```

This name is too generic and it's not consistent with thr rest of the names, please change it for something lkike this
```suggestion
.ris-table2-border-bottom > td {
```